tests/test_dataset.py

```diff
@@ -0,0 +1,253 @@
+import os
+import posixpath
+
+import pytest
+
+import audeer
+import audiofile
+import audb
+import audformat
+import audbcards
+import audplot
+
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+
+
+BUILD = audeer.path('..', 'build', 'html')
+
+
+def test_dataset(cache, tmpdir, db):
+    dataset = audbcards.Dataset(pytest.NAME, pytest.VERSION)
```

Comment:
To make sure we do not use an existing user folder with `Dataset` during the tests as cache, I would change here to

```suggestion
    dataset_cache = audeer.mkdir(audeer.path(tmpdir, 'cache'))
    dataset = audbcards.Dataset(
        pytest.NAME,
        pytest.VERSION,
        cache_root=dataset_cache,
    )
```